helm-charts/charts/kube-starrocks/values.yaml

```diff
@@ -255,8 +255,22 @@ starrocks:
       #   mountPath: /etc/my-configmap
       #   subPath: ""
     # terminationGracePeriodSeconds defines duration in seconds the FE pod needs to terminate gracefully.
+    # default value is 120 seconds
     terminationGracePeriodSeconds: 120
   
+    # Please upgrade the CRD with v1.8.7 released version, if you want to use the following configuration.
+    # including: startupProbeFailureSeconds, livenessProbeFailureSeconds, readinessProbeFailureSeconds
+  
+    # StartupProbeFailureSeconds defines the total failure seconds of startup Probe.
+    # default value is 300 seconds
+    startupProbeFailureSeconds:
+    # LivenessProbeFailureSeconds defines the total failure seconds of liveness Probe.
+    # default value is 15 seconds
+    livenessProbeFailureSeconds:
+    # ReadinessProbeFailureSeconds defines the total failure seconds of readiness Probe.
+    # default value is 15 seconds
+    readinessProbeFailureSeconds:
```

Comment:
is it OK to leave the value empty? looks like weird for an integer type.

Reply:
This is done intentionally. If we set a default value, users with versions <=v1.8.6 would encounter upgrade failures during their next deployment.